model/database.py

```diff
@@ -0,0 +1,38 @@
+from genericpath import exists
+from re import S
+from model.table import *
+
+parent_dir = os.path.abspath(os.path.join(os.path.dirname(__file__), os.pardir))
```

Comment:
Schema should be part of model not in commands, we need to move schema_key to the model and remove this line

Reply:
why remove this line? It is the path to the database directory

Reply:
where do u use this line ?

Reply:
in self.__path , I join this path with the database name

Reply:
make sense